### PR TITLE
[image][android] Add prop - blurRadius

### PR DESCRIPTION
**Pull request by mstach60161**
# Why

Expo image has missing property - `blurRadius`.

# How

- Added prop `blurRadius`.

# Test Plan

Compare with react native image prop.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).

### CODE REVIEW

File: packages/expo-image/android/build.gradle [hunk 1]
```diff
@@ -92,4 +92,5 @@ dependencies {
   api "com.squareup.okhttp3:okhttp:${safeExtGet("okHttpVersion", DEFAULT_OKHTTP_VERSION)}"
 
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${safeExtGet('kotlinVersion', '1.4.21')}"
+  implementation "jp.wasabeef:glide-transformations:4.0.0"
```
Comment (bbarthec) [inline packages/expo-image/android/build.gradle:95]: I see there's a newer version: https://github.com/wasabeef/glide-transformations#gradle. Is there any reason why you choose `4.0.0` over `4.3.0`?
```suggestion
  implementation "jp.wasabeef:glide-transformations:4.3.0"
```

File: packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt [hunk 2]
```diff
@@ -50,6 +51,7 @@ class ExpoImageView(context: ReactContext, private val requestManager: RequestMa
   }
   private var loadedSource: GlideUrl? = null
   internal var sourceMap: ReadableMap? = null
+  private var blurRadius: Int? = null
```
Comment (bbarthec) [inline packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt:54]: Let's convert this to actual property with public setter 💪 
```suggestion
  var blurRadius: Int? = null
    private get
```

File: packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt [hunk 3]
```diff
@@ -62,6 +64,10 @@ class ExpoImageView(context: ReactContext, private val requestManager: RequestMa
     // TODO: repeat mode handling
   }
 
+  internal fun setBlurRadius(blurRadius: Int) {
+    this.blurRadius = blurRadius
+  }
+
```
Comment (bbarthec) [inline packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt:70]: Let's use kotlin properties https://kotlinlang.org/docs/properties.html
```suggestion
```

File: packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewManager.kt [hunk 4]
```diff
@@ -55,6 +55,11 @@ class ExpoImageViewManager(applicationContext: ReactApplicationContext) : Simple
     view.setResizeMode(resizeMode)
   }
 
+  @ReactProp(name = "blurRadius")
+  fun setBlurRadius(view: ExpoImageView, blurRadius: Int) {
+    view.setBlurRadius(blurRadius)
```
Comment (bbarthec) [inline packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewManager.kt:60]:
```suggestion
    view.blurRadius = blurRadius
```

File: packages/expo-image/src/Image.tsx [hunk 5]
```diff
@@ -27,6 +27,7 @@ export interface ImageProps extends AccessibilityProps {
   source?: ImageSourcePropType | null;
   style?: StyleProp<ImageStyle>;
   resizeMode?: ImageResizeMode;
+  blurRadius?: number;
```
Comment (bbarthec) [inline packages/expo-image/src/Image.tsx:30]: Let's annotate it's Android only for now:
```suggestion
  /**
   * @Android only
   */
  blurRadius?: number;
```

**Review by bbarthec — CHANGES_REQUESTED** [review]
Looking promising 💪 
I see that all the properties in the view are written using old Java-styled setters. I believe we should move to reduces boilerplate and leverage Kotlin properties. cc @lukmccall, what do you think?

**Comment by bbarthec** [comment]
`SDK / check-packages` are failing, so you need to do followings:
- `cd packages/expo-image`
- `yarn clean && yarn build`
- `git add . && git commit -m "Rebuild the package"

File: packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt (hunk 3 above)
Comment (mstach60161) [reply packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt:70]: I considered kotlin properties, but finally I followed the existing prop setters. I will change other prop setters in the near future.  👍

**Review by bbarthec — APPROVED** [review]
Looks good 👍 
Resolve conflict/rebuild the build directory and we're ready to merge 💪

**Pull request merged**